testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterReadiness.kt

```diff
@@ -98,6 +99,8 @@ class ClusterReadinessChecker: ClusterReadiness {
         val request = HttpRequest.newBuilder()
             .uri(URI.create(url))
             .build()
-        return client.send(request, HttpResponse.BodyHandlers.ofString())
+        val response = client.send(request, HttpResponse.BodyHandlers.ofString())
+        logger.info("Returning status ${response.statusCode()} $name on $url.")
+        return response
```

Comment:
```suggestion
        return client.send(request, HttpResponse.BodyHandlers.ofString()).also {
           logger.info("Returning status ${response.statusCode()} $name on $url.")
        }
```
slightly more Kotlin idiomatic.